vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/flow-component-renderer.js

```diff
@@ -104,7 +104,11 @@ class FlowComponentRenderer extends PolymerElement {
   }
 
   _attachRenderedComponentIfAble() {
-    if (!this.nodeid || !this.appid) {
+    if (this.appid == null) {
+      return;
+    }
+    if (this.nodeid == null) {
+      this.remove(this.firstChild);
```

Comment:
Use `removeChild` instead of [`remove`](https://developer.mozilla.org/en-US/docs/Web/API/Element/remove) (which removes the element itself, in this case, the `<flow-component-renderer>`, from its parent element). Note that `removeChild` throws if the given parameter is `undefined` (no child element).

Reply:
Done.